apps/userdashboard/assets/js/a4_candy_userdashboard/HoverButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react'
```

Comment:
I'm wondering as this is a general component if it would be better in the contrib folder?